src/types/dataTypes/formatted/banners.ts

```diff
@@ -1,3 +1,17 @@
+export const BannerDisplayType = {
+  PROMO_BANNER: 'promoBanner',
+  FACILITY_BANNER: 'facilityBanner',
+  BANNER: 'banner',
+}
+
+export const BannerTypeMapping = {
+  [BannerDisplayType.PROMO_BANNER]: 'node--promo_banner',
+  [BannerDisplayType.FACILITY_BANNER]: 'node--full_width_banner_alert',
+  [BannerDisplayType.BANNER]: 'node--banner',
+}
```

Comment:
It does feel a little odd, but I'm not sure about a better place... banners are weird case.

The mapping functionality could probably be simplified a bit, but this set of types:
```
export const BannerDisplayType = {
  PROMO_BANNER: 'promoBanner',
  FACILITY_BANNER: 'facilityBanner',
  BANNER: 'banner',
}

export const BannerTypeMapping = {
  [BannerDisplayType.PROMO_BANNER]: 'node--promo_banner',
  [BannerDisplayType.FACILITY_BANNER]: 'node--full_width_banner_alert',
  [BannerDisplayType.BANNER]: 'node--banner',
}
```

is currently needed in two spots:
1. the formatter in `queries/banners.ts`
2. `wrapper/index.tsx` to match the formatted data to a display template

for that reason, i felt putting them in `formatted` made the most sense

Reply:
After looking at this further, I think the reason for needing this in `wrapper/index.tsx` is that the banner `formatter` isn't normalizing the `type` to a non-Drupal naming convention. 

It feels like maybe `BannerTypeMapping` should stay in the query file and then a formatted mapping should be put in the `formatted` type-definition file. Then, in `wrapper/index.tsx`, we'd check for the formatted/normalized type name rather than the Drupal type name.

On a related but separate note, I also find myself wondering if the `BannerDisplayType` definition is necessary. Since the values are not mapped to anything specific, could we just use the prop names directly in `BannerTypeMapping`?
```
export const BannerTypeMapping = {
  PROMO_BANNER: 'node--promo_banner',
  FACILITY_BANNER: 'node--full_width_banner_alert',
  BANNER: 'node--banner',
}
```
With my other suggestion above, we'd then have another mapping definition:
```
export const FormattedBannerTypeMapping = {
  PROMO_BANNER: 'promo_banner',
  FACILITY_BANNER: 'facility_banner',
  BANNER: 'banner',
}
```
Since we are repeating these keys in multiple places, it seems reasonable to consider whether we'd want to pull those keys out to another layer somewhere like we currently have, but I guess I'm not sure I find that to be necessary. I'm not sure it saves us from any future pain. If we ever would have to change the key `PROMO_BANNER`, for instance, then we'd almost certainly need to change `BannerDisplayType.PROMO_BANNER` anyway.